examples/time_slider.html

```diff
@@ -26,17 +26,17 @@
         left: 40px;
       }
     </style>
-    <link rel="stylesheet" href="http://libs.cartocdn.com/cartodb.js/v3/3.15/themes/css/cartodb.css" />
+    <link rel="stylesheet" href="https://libs.cartocdn.com/cartodb.js/v3/3.15/themes/css/cartodb.css" />
     <link rel="stylesheet" href="http://code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css" />
```

Comment:
https://code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css seems to work fine